spec/sql_footprint_spec.rb

```diff
@@ -6,63 +6,73 @@
   end
 
   describe '.start' do
-    let!(:logger) { described_class.start }
-    let(:sql) { logger.logs.last }
+    before do
+      described_class.start
 
-    it 'logs sql' do
-      Widget.create!
-      expect(logger.logs.length).to eq 1
+      # Warmup so Widget.create just makes one new flavor of SQL
+      Widget.first
+      Cog.create!
     end
 
-    it 'logs exists sql' do
-      Widget.exists? name: 'a'
-      expect(logger.logs.length).to eq 1
+    it 'logs sql' do
+      expect { Widget.create! }.to change { described_class.lines.length }.by(+1)
```

Comment:
Did you mean to have `+1` instead of `1` here? And if you did, I'd love to learn the advantage over the latter.